nevergrad/functions/causaldiscovery/core.py

```diff
@@ -0,0 +1,89 @@
+import typing as tp
+import networkx as nx
+import cdt
+from cdt.metrics import (precision_recall, SHD)
+import nevergrad as ng
+from ..base import ExperimentFunction
+
+
+# pylint: disable=too-many-instance-attributes,too-many-arguments,too-many-statements,too-many-locals
```

Comment:
```suggestion
# pylint: disable=too-many-instance-attributes,too-many-arguments,too-many-locals
```
I do not enforce pylint strictly but indeed I use it on my laptop. Those tags deactivate some of the issues, but I doubt you would get too-many-statements here